lib/config.sh

```diff
@@ -1,12 +1,19 @@
 #! /bin/bash
```

Comment:
do we want this in _lib/_? What divides the content of _lib/_ vs. _aws/_ and _gce/_?